onoff.d.ts

```diff
@@ -0,0 +1,46 @@
+export type High = 1;
+export type Low = 0;
+export type Directions = "in" | "out" | "high" | "low";
```

Comment:
`Directions` should be renamed to `Direction`. i.e., use the singular rather than the plural form.